src/content/docs/guides/i18n.mdx

```diff
@@ -56,17 +56,68 @@ You can also visit GitHub directly and go through your language's [open Pull Req
+The docs repository contains two different kinds of translatable content: docs pages and UI strings. The following file tree shows where to find these in the repository.
 
-We also have UI text, generally consisting of relatively short bits of text used to label or structure components of the documentation UI. For example, our auto-generated table of contents has a heading that in English reads “On this page.” Those texts are stored separately from the page content itself, which means we have to translate them separately.
+import { FileTree } from '@astrojs/starlight/components';
 
-These labels live in the `src/i18n` folder, with a directory for each language similar to how pages work. Unlike pages, these translations look more like a dictionary, mapping standard keys to translated strings. Each language should provide the following files:
+<FileTree>
 
-- `nav.ts` — translates the labels for the navigation menu
-- `ui.ts` — translates miscellaneous bits of text found around the docs
-- `docsearch.ts` — translates the search component
+- ...
+- src/
+  - ...
+  - content/
+    - docs/ MDX page content, one directory per language
+      - ar/
+      - de/
+      - en/
+      - ...
+    - i18n/ UI string translations, one file per language
+      - ar.yml
+      - de.yml
+      - en.yml
+      - ...
+  - ...
+- astro.config.ts
+- package.json
+- ...
 
-See [`src/i18n/es`](https://github.com/withastro/docs/tree/main/src/i18n/es) as an example of these three files.
+</FileTree>
+
+#### Docs pages
+
+Each documentation page lives in the `src/content/docs/` directory of the docs <abbr title="repository">repo</abbr>. There you’ll find a directory for each currently supported language. Each page is an MDX file to support rich text formatting and Astro components.
+
+For example, the English language “Getting Started” page is at `src/content/docs/en/getting-started.mdx` and the same page in French is at `src/content/docs/fr/getting-started.mdx`.
+
+#### Reusable UI strings
+
+We also have translations for UI text strings. These are short bits of text used to label or structure components of the documentation UI. For example, the text “Sponsored by” before our sponsor logos on the English pages is translated so we can show “Sponsorisé par” on the French pages instead.
+
+UI strings are stored in the `src/content/i18n/` folder, with a dedicated YAML file for each language. Unlike pages, these translations look more like a dictionary, mapping standard keys to translated strings. Comments can be included on a line starting with a `#`.
+
+The following example shows an abbreviated version of the English UI strings:
+
+```yml title="src/content/i18n/en.yml"
+a11y.sectionLink: Section titled
+# Site settings
+site.title: Astro Documentation
+# Left Sidebar
+leftSidebar.sponsoredBy: Sponsored by
+# ...
+```
+
+Keys are not translated, only the English text. So the example above is translated into French is:
```

Comment:
@sarah11918 Aren’t the keys also in English? 😅

Reply:
Maybe something like this to further clarify?

```suggestion
Keys, although they are written in English, should not be translated. The only part that should be translated is the text following the key. For example, when translated into French, it appears like this:
```

Reply:
I like that! But I do think it’s a bit much when just explaining a the example. We can probably simplify back to something like the original:
```suggestion
Keys are not translated, so the example above is translated into French as:
```